test-app/tests/integration/components/prismic/element-test.js

```diff
@@ -22,11 +22,13 @@ module('Integration | Component | prismic/element', function (hooks) {
       };
       this.onUnknownTag = (node) => {
         assert.deepEqual(node, this.node);
```

Comment:
Do you still need this `assert`? To my understanding, this assertion was just added to make sure `onUnknownTag` was called

Reply:
Ah you’re right! I’ve removed it 👍